### PR TITLE
Ignite 944 validation to check for non existing lines

### PR DESCRIPTION
**Pull request by samaitra**

### CODE REVIEW

**Comment by samaitra** [comment]
@dmagda 

Hey Denis,

Can you please review this change for validation of non existing lines.

Regards
Saikat

File: modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/log/GridLogCommandHandler.java [hunk 1]
```diff
@@ -161,6 +161,10 @@ private String readLog(int from, int to, File logFile) throws IgniteCheckedExcep
 
                 start++;
             }
+
+            if (start < from || start > (to + 1)){
```
Comment (dmagda) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/log/GridLogCommandHandler.java]: I think that you can simply do the following check 

if (content.size() == 0)
thrown new IgniteCheckedException(...)

Comment (samaitra) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/rest/handlers/log/GridLogCommandHandler.java]: yes, makes sense. Thanks !!!

**Comment by samaitra** [comment]
@dmagda 

Thank you Denis. I have updated the files as per review comments.

Regards
Saikat

**Comment by samaitra** [comment]
@dmagda 

Can you please review and share feedback.

Regards
Saikat

**Pull request closed without merging**